tools/ci.psm1

```diff
@@ -225,18 +225,19 @@ function Invoke-AppVeyorInstall
 
         # Provide credentials globally for remote tests.
         $ss = ConvertTo-SecureString -String $password -AsPlainText -Force
-        $appveyorRemoteCredential = [PSCredential]::new("$env:COMPUTERNAME\$userName", $ss)
-	    $appveyorRemoteCredential | Export-Clixml -Path "$env:TEMP\AppVeyorRemoteCred.xml" -Force
+        $ciRemoteCredential = [PSCredential]::new("$env:COMPUTERNAME\$userName", $ss)
+	$ciRemoteCredential | Export-Clixml -Path "$env:TEMP\AppVeyorRemoteCred.xml" -Force
```

Comment:
Indentation does not look right.

Reply:
Will resolve when I get back home.  Thanks for noticing.

Reply:
Thanks.  I believe it is fixed now :)